code/datums/abilities/flock/flockmind.dm

```diff
@@ -57,10 +57,38 @@
 
 /datum/targetable/flockmindAbility/spawnEgg/cast(atom/target)
 	if(..())
-		return 1
+		return
```

Comment:
I think returning 1 (or TRUE) indicates a fail state for abilities

Reply:
Yeah, the parent function is this I think
![image](https://user-images.githubusercontent.com/53062374/158508941-38b5cec6-dded-45be-88b5-9424eba01164.png)

Reply:
> I think returning 1 (or TRUE) indicates a fail state for abilities

I just removed it because there was no cooldown with the ability, but I can add it back if that's better.

Reply:
Always use TRUE